src/robot/ros/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+#==============================================================================
```

Comment:
I might have missed where this is called from, but why do we need a separate CMakeLists over here?

Reply:
See control/ros for an example. This is here to define a new component robot::Ros, which holds all the Ros-dependent functionality. Right now, that's just URDF/srdfdom, but will contain controller switching in the future.

This should be called from robot CMakeLists as add_subdirectory("ros")